.env.docker

```diff
@@ -89,6 +89,7 @@ WEBHOOK_BASIC_AUTH_PASSWORD=cCVMxdX8hfctDefk
 GENERATED_USERS_FILE_PATH=var/user-files/
 
 FEATURE_GENERATE_USERS_ON_LINEITEM_UPDATED_ENABLED=1
+FEATURE_GENERATED_USERS_S3_FOLDER_SYNC_ON_LINEITEM_UPDATED_ENABLED=0
```

Comment:
for consistency, I think this should either be FEATURE_GENERATE_*** or FEATURE_GENERATED_*** for both variables

Reply:
FEATURE_GENERATE_***  is used to enable whether we needs to generate users during remote publishing webhook
FEATURE_GENERATED_***  is used to to decide whether the generated users needs to save to the S3

So I think we needs to use FEATURE_GENERATE_*** and  FEATURE_GENERATED_** variable name